src/profile-logic/data-structures.js

```diff
@@ -272,7 +285,8 @@ export function getEmptyThread(overrides?: $Shape<Thread>): Thread {
     name: 'Empty',
     pid: 0,
     tid: 0,
-    samples: getEmptySamplesTable(),
+    // Creating samples with event delay since it's the new samples table.
+    samples: getEmptySamplesTableWithEventDelay(),
```

Comment:
I didn't create two versions of `getEmptyThread` because this is mostly used by tests(and an exception of chrome import, which we are okay changing with). So this looks okay to change to eventDelay directly.